docs/pages/result.rst

```diff
@@ -116,6 +116,28 @@ If you want to `safe` handle only a set of exceptions:
     ...
   ValueError: Too big
 
+attempt
+~~~~~~~
+
+Similar to :func:`safe <returns.result.safe>` function but instead
+of wrapping the exception error in a Failure container it'll wrap the
```

Comment:
```suggestion
of wrapping the exception error in a :func:`~Failure` container it'll wrap the
```

Reply:
I've added a class reference since the exposed `Failure` is a class:
```text
:class:`Failure <returns.result.Failure>`
```